src/views/Overview/Home.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <b-container fluid>
+    <h1>Overview</h1>
+    <b-row>
+      <b-col lg="8" sm="12">
+        <section>
+          <h2>Server Information</h2>
+          <b-row>
+            <b-col sm="6">
+              <dl>
+                <dt>MODEL</dt>
+                <dd>{{ system.Model || "N/A" }}</dd>
+              </dl>
+            </b-col>
+            <b-col sm="6">
+              <dl>
+                <dt>MANUFACTURER</dt>
+                <dd>{{ system.Manufacturer || "N/A" }}</dd>
+              </dl>
+            </b-col>
+            <b-col sm="6">
+              <dl>
+                <dt>SERIAL NUMBER</dt>
+                <dd>{{ system.SerialNumber || "N/A" }}</dd>
+              </dl>
+            </b-col>
+            <b-col sm="6">
+              <dl>
+                <dt>FIRMWARE VERSION</dt>
+                <dd>{{ software.Version || N/A }}</dd>
```

Comment:
Does "N/A" need to be wrapped in quotes?

Reply:
without quotes it shows "NaN" in the view

Reply:
done